src/olympia/zadmin/models.py

```diff
@@ -15,20 +20,21 @@ class Meta:
     def __str__(self):
         return self.key
 
-    @property
-    def json(self):
-        try:
-            return json.loads(self.value)
-        except (TypeError, ValueError):
-            return {}
 
-
-def get_config(conf, default=None, *, json_value=False):
+def get_config(key, default=None, *, json_value=False, int_value=False):
```

Comment:
I made that change on purpose to get an explicitly "null" value - I wanted the value to potentially be unsafe so that we'd have to explicitly care about that case - I prefer failing loudly.

Otherwise, and especially for `int_value`s, it may be a little dangerous, depending on what the caller does with the value - if it's a threshold or a ceiling for instance, `0` might have some unwanted consequences...

Reply:
Mmm, I guess that's fine, if all the existing usage of `get_config` provides a default.